src/Umbraco.Web.UI.Client/src/views/common/dialogs/login.html

```diff
@@ -40,7 +40,10 @@
                 </div>
 
                 <div class="control-group" ng-class="{error: loginForm.password.$invalid}">
-                    <input type="password" ng-model="password" name="password" class="input-xlarge" localize="placeholder" placeholder="@placeholders_password" autocomplete="off" />
+                    <input type="{{showpassword ? 'text' : 'password'}}" ng-model="password" name="password" class="input-xlarge password" ng-class="{'has-password': password.length > 0}" localize="placeholder" placeholder="@placeholders_password" autocomplete="off" />

```

Comment:
@Shazwazza I just saw this example using a directive:
http://angularscript.com/angular-toggle-password-directive/

demo: http://codepen.io/jakob-e/pen/eNBQaP